sass/components/_footer.scss

```diff
@@ -0,0 +1,38 @@
+// bottom sticky footer
+//
+// a few things to note:
+// 1. Avoid the IE 10-11 `min-height` bug.
+// 2. Set `flex-shrink` to `0` to prevent some browsers from
+//    letting these items shrink to smaller than their content's default
+//    minimum size. See http://bit.ly/1Mn35US for details.
+// 3. Use `%` instead of `vh` since `vh` is buggy in older mobile Safari.
+//
+// scss-lint:disable IdSelector
+
+html,
+body,
+#app {
+  height: 100%;
+}
+
+.site {
+  display: flex;
+  flex-direction: column;
+  height: 100%; // 1, 3
+}
+
+.footer {
```

Comment:
that's fine -- i was thinking we could potentially have other footers in certain contexts that shouldn't be sticky, hence this qualifier, but i'm ok just targeting the `footer` element for now